submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+/* NAV START */
+.container {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 340px;
```

Comment:
Your styles should take into account changes in the content so that in the future there will be no problems with the display of new content.

Reply:
@A-Ostrovnyy do you mean? what exactly wrong (nor correct) with these four lines of code? How can I improve it?

Reply:
I mean all your navigation block is 340px width when I add a couple nav__item's it will be looking not good.

Reply:
@A-Ostrovnyy Done, thank you a lot for the explanation, I have changed the width to 100%